### PR TITLE
WIP: Added page for documentation for Monzo platform

### PR DESCRIPTION
**Pull request by TomAFrench**
**Description:**

Added page for documentation for Monzo platform

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#17218

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.monzo.markdown [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+description: "How to integrate your Monzo Bank account within Home Assistant."
+date: 2018-09-07 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: "monzobank.png"
+ha_category: Finance
+featured: false
+ha_release: "0.79"
```
Comment (WofWca) [inline source/_components/sensor.monzo.markdown]: This will have to be `"0.80"`.

**Review by WofWca — CHANGES_REQUESTED** [review]

File: source/_components/sensor.monzo.markdown (hunk 1 above)
Comment (TomAFrench) [reply source/_components/sensor.monzo.markdown]: Fixed in new commit. Thanks.

Comment (frenck) [reply source/_components/sensor.monzo.markdown]: Not correct, @Solije @WofWca.
The "0.80" already has been cut off (feature stop/beta).
So the first possible one for this to get into would be `0.81`

**Review by frenck — APPROVED** [review]
Overall, it looks good @Solije! Thanks 👍 

✅ Can be merged as soon as the parent PR passes and no further changes to the docs are needed.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
@Solije I know this is WIP, nevertheless, the basics are covered and checked now.

**Comment by frenck** [comment]
Closing this PR, since the parent PR has been closed.

**Pull request closed without merging**